### PR TITLE
Add decorator for linked pictures

### PR DESCRIPTION
**Pull request by alexcarol**
This is required as a workaround for sharepoint word limitations

Brought and adapted from: https://github.com/adobe/express-website/blob/main/express/scripts/scripts.js#L1707

Needed for #3 

Test URLs:
- Before: https://main--eder-group--hlxsites.hlx.page/
- After: https://linked-images-doc--eder-group--hlxsites.hlx.page/

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -40,6 +40,27 @@ function buildAutoBlocks(main) {
...
+    const a = nextSib.querySelector('a');
+    if (a && a.textContent.trim().startsWith('https://')) {
+      a.innerHTML = '';
+      a.className = '';
+      a.appendChild(picture);
+    }
+  }
+}
+
+/* workaround for word not being able to add links to pictures */
```
Comment (mhaack) [inline scripts/scripts.js:55]: Don't call it workaround :-) as this is the default behaviour when using word.

File: scripts/scripts.js [hunk 2]
```diff
@@ -40,6 +40,27 @@ function buildAutoBlocks(main) {
   }
 }
 
+function linkPicture(picture) {
```
Comment (mhaack) [inline scripts/scripts.js:43]: Can this interferer with `decorateButtons` ?

**Review by mhaack — APPROVED** [review]

File: scripts/scripts.js (hunk 2 above)
Comment (alexcarol) [reply scripts/scripts.js:43]: It won't as long as it's decorated after decorateButtons (where it is now). We could change the code to be more specific, but this is good enough for our needs.

File: scripts/scripts.js (hunk 1 above)
Comment (alexcarol) [reply scripts/scripts.js]: Updated

**Review by mhaack — APPROVED** [review]

**Pull request merged**